runtime/drivers/admin/admin.go

```diff
@@ -125,6 +125,8 @@ type Handle struct {
 
 	// downloadURL is set when using one-time uploads
 	downloadURL string
+	assetID     string
+	createdOn   time.Time
```

Comment:
Similar to the API signature, perhaps adding `archive` prefixes here might also be helpful to disambiguate